main.tf

```diff
@@ -41,9 +41,9 @@ locals {
   xchain_indexer_staging_docker_image = "omniops/xchain-indexer:0.1.3"
   blockscout_staging_docker_image = "omniops/blockscout:0.1.0.commit.2404d446"
 
-  omni_testnet_ws = "ws://testnet-sentry-explorer.omni.network:8546"
+  omni_testnet_ws = "ws://${var.testnet_sentry_node_url}:8546"
```

Comment:
Should we treat the ports in the same way. Maybe not needed for testnet atm.

Reply:
I think it's okay to leave ports in they are standard